### PR TITLE
fix: proxy factory impl names

### PR DESCRIPTION
**Pull request by paulbalaji**
Fixes issue where contract names we've configured in the deploy script do not match what they actually are in the current contracts

This surfaces up as an issue when attempting to verify these factories (cc @tkporter)

### CODE REVIEW

File: typescript/sdk/src/deploy/contracts.ts [hunk 1]
```diff
@@ -21,9 +22,9 @@ type ProxyFactoryImplementations = Record<keyof ProxyFactoryFactories, string>;
...
 // must match contract names for verification
 export const proxyFactoryImplementations: ProxyFactoryImplementations = {
-  merkleRootMultisigIsmFactory: 'StaticMerkleRootMultisigIsm',
-  messageIdMultisigIsmFactory: 'StaticMessageIdMultisigIsm',
-  aggregationIsmFactory: 'StaticAggregationIsm',
-  aggregationHookFactory: 'StaticAggregationHook',
-  routingIsmFactory: 'DomaingRoutingIsm',
+  staticMerkleRootMultisigIsmFactory: 'StaticMerkleRootMultisigIsm',
+  staticMessageIdMultisigIsmFactory: 'StaticMessageIdMultisigIsm',
+  staticAggregationIsmFactory: 'StaticAggregationIsm',
```
Comment (paulbalaji) [inline typescript/sdk/src/deploy/contracts.ts:27]: these three aren't in the `buildArtifact.json` for some reason

Comment (paulbalaji) [reply typescript/sdk/src/deploy/contracts.ts:27]: fake news - stale artifact

File: typescript/sdk/src/deploy/contracts.ts
Comment (yorhodes) [inline typescript/sdk/src/deploy/contracts.ts:1]: thanks for doing this

File: typescript/cli/examples/contract-artifacts.yaml
Comment (yorhodes) [inline typescript/cli/examples/contract-artifacts.yaml:1]: how can we make sure this doesnt go stale?

**Review by yorhodes — APPROVED** [review]

File: typescript/cli/examples/contract-artifacts.yaml
Comment (paulbalaji) [reply typescript/cli/examples/contract-artifacts.yaml:1]: not sure, but I can add a comment to the relevant part to let folks know they should update the example

**Pull request merged**